packages/material-ui/src/Select/SelectInput.js

```diff
@@ -285,7 +285,7 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
         aria-haspopup="listbox"
         aria-owns={open ? `menu-${name || ''}` : undefined}
         onKeyDown={handleKeyDown}
-        onClick={disabled || readOnly ? null : handleClick}
+        onMouseDown={disabled || readOnly ? null : handleClick}
```

Comment:
We want to make sure that any forwarding logic (`onClick` in `handleclick`) is still forwarded properly. So any open logic should be moved into an `handleMouseDown` while the old `handleClick` no longer opens/closes the options.

When we do this we need need to make sure Space and Enter still open the options in their handleClick

Reply:
Understood. Thanks for taking the time to review this.